src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -207,44 +207,14 @@ void GraphingCalculator::OnEquationsVectorChanged(IObservableVector<EquationView
     GraphingControl->PlotGraph(false);
 }
 
-wstringstream GraphingCalculator::FormatTraceValue(double min, double max, float pointValue)
-{
-    wstringstream traceValueString;
-
-    // Extract precision we will round to
-    auto precision = static_cast<int>(floor(log10(max - min)) - 3);
-
-    // Determine if we want to show scientific notation instead
-    if (precision <= -7 || precision >= 7)
-    {
-        traceValueString << scientific;
-    }
-    else
-    {
-        traceValueString << fixed;
-    }
-
-    // If we are rounding to a decimal place, set the precision
-    if (precision < 0)
-    {
-        traceValueString << setprecision(::min(7, abs(precision))) << pointValue;
-    }
-    else
-    {
-        traceValueString << setprecision(0) << pointValue;
-    }
-
-    return traceValueString;
-}
-
 void GraphingCalculator::OnTracePointChanged(Point newPoint)
 {
     wstringstream traceValueString;
 
     double xAxisMin, xAxisMax, yAxisMin, yAxisMax;
     GraphingControl->GetDisplayRanges(&xAxisMin, &xAxisMax, &yAxisMin, &yAxisMax);
-
-    traceValueString << "(" << FormatTraceValue(xAxisMin, xAxisMax, newPoint.X).str() << ", " << FormatTraceValue(yAxisMin, yAxisMax, newPoint.Y).str() << ")";
+    
+    traceValueString << "(" << newPoint.X << ", " << newPoint.Y << ")";    
```

Comment:
Another change occurs here when converting the float to a string, in the same scenario as my other comment the y value goes from 123703.617 to 123704. I think to fix this we may want to call SetPrecision and set it to the maximum amount of decimals allowed and perhaps run the string through TrimTrailingZeros in NumberFormattingUtils.

Reply:
Great catch, float was introducing some unnecessary rounding, pushed changed to convert to use doubles instead of built-in point class.